runelite-client/src/test/java/net/runelite/client/plugins/loottracker/LootTrackerPluginTest.java

```diff
@@ -160,6 +160,27 @@ public void testPickPocket()
 		assertEquals(LootRecordType.PICKPOCKET, lootTrackerPlugin.lootRecordType);
 	}
 
+	@Test
+	public void testPickPocketToSeedBox()
+	{
+		when(client.getBoostedSkillLevel(Skill.THIEVING)).thenReturn(42);
+
+		final ItemPrice seedPrice = new ItemPrice();
+		seedPrice.setId(ItemID.POTATO_SEED);
+		seedPrice.setName("Potato seed");
+		when(itemManager.search("Potato seed")).thenReturn(Collections.singletonList(seedPrice));
+
+		LootTrackerPlugin lootTrackerPluginSpy = spy(this.lootTrackerPlugin);
+		doNothing().when(lootTrackerPluginSpy).addLoot(any(), anyInt(), any(), any(), any());
+
+		ChatMessage chatMessage = new ChatMessage(null, ChatMessageType.SPAM, "", "The following stolen loot gets added to your seed box: Potato seed x 3.", "", 0);
+		lootTrackerPluginSpy.onChatMessage(chatMessage);
+
+		verify(lootTrackerPluginSpy).addLoot(null, -1, LootRecordType.PICKPOCKET, 42, Arrays.asList(
+				new ItemStack(ItemID.POTATO_SEED, 3, null)
```

Comment:
This is still indented one level too far. (I'll commit this via GitHub's UI)
```suggestion
			new ItemStack(ItemID.POTATO_SEED, 3, null)
```